extension/src/sites/manage.js

```diff
@@ -771,6 +771,12 @@ async function handleSettings (hasChanged = false) {
     this.disabled = true
     this.innerText = 'Reinstalling...'
 
+    // Listen for use Linked runtime
+    document.getElementById('settings-use-linked-runtime').addEventListener('change', async function () {
+      config.use_linked_runtime = this.checked
+      await setConfig(config)
+    })
```

Comment:
For the extension settings, you can check how settings like `settings-always-patch` are implemented. So, this code should be inside `setTimeout(async () => {` (around line 875).